cpp/include/raft/spatial/knn/ann.cuh

```diff
@@ -64,24 +58,22 @@ inline void approx_knn_build_index(raft::handle_t& handle,
  *                       their query point
  * @param[out] indices indices of the nearest neighbors
  * @param[in] index index to perform a search with
+ * @param[in] params parameters used to build the index
  * @param[in] k the number of nearest neighbors to search for
  * @param[in] query_array the query to perform a search with
```

Comment:
Please describe the memory layout of the input / output arrays (device vs host, row or column major).